include/iceflow/consumer-tlv.hpp

```diff
@@ -520,7 +559,7 @@ class ConsumerTlv {
     int difference = 0;
     int dataCount = 0;
   };
-  // lowerSeqnum, stream, diff, available data
+  // lower Seq Num, stream, diff,  data count
```

Comment:
Should probably be a proper Doxygen comment.

See an initial proposal below:

```suggestion
  /**
   * Maps pairs of lower sequence numbers and stream numbers to
   * data counts.
   */
```